demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -255,8 +254,6 @@ def execute_update(self) -> bool:
         changed_files = {}
         for packfile in self.modified_files_in_pack:
             file_name, file_type = self.get_changed_file_name_and_type(packfile)
-            if file_type in SKIP_RELEASE_NOTES_FOR_TYPES:
```

Comment:
why deleted?

Reply:
The skip is implemented in hight functions.